src/components/ItemEditorPage/LeftPanel/LeftPanel.container.tsx

```diff
@@ -0,0 +1,112 @@
+import { fetchItemsRequest, fetchOrphanItemRequest, FETCH_ITEMS_REQUEST, FETCH_ORPHAN_ITEM_REQUEST } from 'modules/item/actions'
+import { LeftPanelContainerProps } from './LeftPanel.types'
+import LeftPanel from './LeftPanel'
+
+const LeftPanelContainer: React.FC<LeftPanelContainerProps> = props => {
+  const dispatch = useDispatch()
+  const selectedCollectionId = useGetSelectedCollectionIdFromCurrentUrl()
+  const selectedItemId = useGetSelectedItemIdFromCurrentUrl()
+  const isReviewing = useGetIsReviewingFromCurrentUrl()
+  const visibleItems = useGetVisibleItems()
+
+  const address = useSelector((state: RootState) => getAddress(state))
+  const isConnectedValue = useSelector((state: RootState) => isConnected(state))
+  const items = useSelector((state: RootState) => getItems(state))
+  const collections = useSelector((state: RootState) => getAuthorizedCollections(state))
+  const bodyShape = useSelector((state: RootState) => getBodyShape(state))
+  const wearableController = useSelector((state: RootState) => getWearablePreviewController(state))
+  const isPlayingEmoteValue = useSelector((state: RootState) => isPlayingEmote(state))
+  const hasUserOrphanItemsValue = useSelector((state: RootState) => hasUserOrphanItems(state))
+  const loading = useSelector((state: RootState) => getLoading(state))
+
+  const selectedItem = useSelector((state: RootState) => (selectedItemId ? getItem(state, selectedItemId) : null))
+  const itemsPaginationData = useSelector((state: RootState) => {
+    if (selectedCollectionId) {
+      return getPaginationData(state, selectedCollectionId)
+    }
+    if (address) {
+      return getPaginationData(state, address)
+    }
+    return undefined
+  })
+  const collectionsPaginationData = useSelector((state: RootState) =>
+    !selectedCollectionId ? getCollectionsPaginationData(state) : undefined
+  )
+  const orphanItems = useSelector((state: RootState) => {
+    if (address && !selectedCollectionId) {
+      return getPaginatedWalletOrphanItems(state, address, itemsPaginationData?.limit)
+    }
+    return getWalletOrphanItems(state)
+  })
+  const isLoading = useMemo(
+    () => isLoadingType(loading, FETCH_ITEMS_REQUEST) || isLoadingType(loading, FETCH_ORPHAN_ITEM_REQUEST),
+    [loading]
+  )
+
+  const onSetItems: ActionFunction<typeof setItems> = useCallback(items => dispatch(setItems(items)), [dispatch])
+  const onFetchCollections: ActionFunction<typeof fetchCollectionsRequest> = useCallback(
+    (address, params) => dispatch(fetchCollectionsRequest(address, params)),
+    [dispatch]
+  )
+  const onFetchOrphanItems: ActionFunction<typeof fetchItemsRequest> = useCallback(
+    (address, params) => dispatch(fetchItemsRequest(address, params)),
+    [dispatch]
+  )
+  const onFetchOrphanItem: ActionFunction<typeof fetchOrphanItemRequest> = useCallback(
+    address => dispatch(fetchOrphanItemRequest(address)),
+    [dispatch]
+  )
+
+  return (
+    <LeftPanel
+      {...props}
```

Comment:
```suggestion
```

Reply:
The LeftPanel has some props that are passed through the container into the component, that's why I'm adding the original props to it. There are some props that will get overwritten by the other props defined later.